lib/IRGen/IRGenModule.cpp

```diff
@@ -693,6 +693,7 @@ void IRGenModule::emitAutolinkInfo() {
     }
     break;
   }
+  case llvm::Triple::COFF:  // for cygwin
```

Comment:
Is the section name `.swift1_autolink_entries` too long for COFF?  How does autolinking work?